groupStatusesWriter.php

```diff
@@ -211,7 +217,17 @@ function get_job_status($results, $jobName){
 
         foreach($block as $index => $value) {
             $valStat = get_job_status($jobs,$value);
-            $jobList .= "\t\t{\"name\": \"" . $value . "\", \"culprit\": \"" . get_status_culprit($value) . "\", \"status\": \"" . $valStat . "\", \"group\": \"" . $key . "\", \"subtabid\": \"". $count . $subcount .  "\"},\n";
+             
+            if($valStat == "failed" || $valStat == "run_fail") {
+                 $jobListRed .= "\t\t{\"name\": \"" . $value . "\", \"culprit\": \"" . get_status_culprit($value) . "\", \"status\": \"" . $valStat . "\", \"group\": \"" . $key . "\", \"subtabid\": \"". $count . $subcount .  "\"},\n";
+            }
+            else if($valStat == "passed" || $valStat == "run_pass") {
```

Comment:
minor, but there is a method called elseif which should be used here.

http://stackoverflow.com/questions/3662412/are-elseif-and-else-if-completely-synonymous